File: javalib/src/main/java/com/read/javalib/MainActivity.java
package com.read.javalib;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.View;

import com.read.javalib.generic.GenericTest;

/**
 * Author: 陈李冠
 * Version: 1.0.0
 * Date: 2020/4/1
 * Mender:
 * Modify:
 * Description: java测试
 */
public class MainActivity extends AppCompatActivity {

    private static final String TAG = "javalib + MainActivity";

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    /**
     * 泛型测试
     *
     * @param view
     */
    public void textGeneric(View view) {
        GenericTest genericTest = new GenericTest();
        genericTest.genericErasure();
    }
}

File: kotlinlib/src/main/java/com/read/javalib/MainActivity.kt
package com.read.javalib

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity;
import android.view.View
import com.read.javalib.generic.GenericTest

/**
 * Author: 陈李冠
 * Version: 1.0.0
 * Date: 2020/4/1
 * Mender:
 * Modify:
 * Description: 测试kotlin
 */
class MainActivity : AppCompatActivity() {

    companion object {
        private const val TAG = "javalib + MainActivity"
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    /**
     * 泛型测试
     *
     * @param view
     */
    fun textGeneric(view: View?) {
        val genericTest = GenericTest()
        genericTest.genericErasure()
    }
}